dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/data/file/ImmediateFileWriterTest.kt

```diff
@@ -189,6 +189,24 @@ internal class ImmediateFileWriterTest {
         verifyZeroInteractions(mockDeferredHandler)
     }
 
+    @Test
+    fun `𝕄 do nothing 𝕎 write() and FileOrchestrator returns a file that doesn't exist`(
+        @StringForgery dirName: String,
+        @StringForgery fileName: String,
+        forge: Forge
+    ) {
+        val nonExistentDir = File(tempRootDir, dirName)
+        val file = File(nonExistentDir, fileName)
+        val modelValue = forge.anAlphabeticalString()
```

Comment:
fakeModelValue

Reply:
This naming scheme is for field usually. In a method that can become very verbose don't you think ?

Reply:
it might get at some point, then let's make it optional but I guess you should update the `Contributing.md` file in this case.